org.gnome.Totem.json

```diff
@@ -51,7 +48,8 @@
         /* im-status plugin */
         "--talk-name=org.gnome.SessionManager",
         /* Codecs extension */
-        "--env=GST_PLUGIN_PATH=/app/lib/codecs/lib/gstreamer-1.0"
+        "--env=GST_PLUGIN_PATH=/app/lib/codecs/lib/gstreamer-1.0",
+        "--metadata=X-DConf=migrate-path=/org/gnome/Totem/"
```

Comment:
Can you please comment this line and move it back to where the dconf hole punch was?

Looks good otherwise, thanks!